src/types/FooterProps.ts

```diff
@@ -0,0 +1,9 @@
+import { Todo } from './Todo';
+import { Filter } from './Filter';
+
+export interface FooterProps {
+  todos: Todo[];
+  filterBy: string;
+  setFilterBy: (filterBy: Filter) => void;
+  handleClearCompleted: () => void;
+}
```

Comment:
it's redundant to create separate files for props, better to keep them close to the component

Reply:
Sure. It's way more easier to and new props or change existing now.